src/njoy21/input/THERMR/Card2/Nbin.hpp

```diff
@@ -0,0 +1,18 @@
+struct Nbin {
+  using Value_t = int;
+  static std::string name(){ return "nbin"; }
+
+  static std::string description(){
+    return
+      "nbin is the number of equi-probable scattering angles available. For\n"
```

Comment:
This needs to be reworded a bit. It sounds as if it is copied out of the manual, but doesn't include enough information; i.e., why 8?